### ISSUE TITLE
Problem with SWIG binding when accessing NeighMoving.getBipts()

### ISSUE THREAD

**Issue by nicolas-clausolles** [gstlearn/gstlearn#133]
### To reproduce :
```python
import gstlearn as gl

neigh = gl.NeighMoving()
screen = gl.PolyLine2D([0, 1], [0, 1])
faults = gl.Faults()
faults.addFault(screen)
target = gl.BiTargetCheckFaults(faults)
neigh.addBiTargetCheck(target)
print(neigh)  # Ok
targets = neigh.getBipts()  # Fails
```

Error message:
```
Moving Neighborhood
===================
Minimum number of samples           = 1
Maximum number of samples           = 1000
- Separated by Faults

swig/python detected a memory leak of type 'std::vector< ABiTargetCheck *,std::allocator< ABiTargetCheck * > > *', no destructor found.
```

### A potential solution?
There is probbly a missing reference `&` in the return type of [`NeighMoving::getBipts()`](https://github.com/gstlearn/gstlearn/blob/324b9c2920c30b74215f361c68c362530a7e8ec7/include/Neigh/NeighMoving.hpp#L83). I assume it should be 
```const std::vector<ABiTargetCheck*>& getBipts() const { return _bipts; }```?

**Comment by drenard77810** [comment]
You are right.

**Comment by fabien-ors** [comment]
Fixed in 95159ff
PR #137 currently under validation
